reccmp/isledecomp/compare/core.py

```diff
@@ -199,11 +204,11 @@ def _load_cvdump(self):
                 if string_info.is_utf16:
                     continue
 
-                raw = self.recomp_bin.read(addr, sym.size())
+                size = sym.size()
+                assert size is not None
 
-                # read returns None when reading 0 bytes
```

Comment:
This comment was actually wrong - I added a unit test to verify that reading 0 bytes returns `b""`